.travis.yml

```diff
@@ -15,30 +15,31 @@ go:
   - 1.x
   - 1.13.x
```

Comment:
in the latest version of go-tool, the version 1.13 is no longer supported

https://github.com/octomation/go-tool/blob/master/.travis.yml#L13-L17